gradle/dependency-management/agp-versions.properties

```diff
@@ -1,4 +1,4 @@
 # Generated - Update by running `./gradlew updateAgpVersions`
-latests=7.4.2,8.0.2,8.2.2,8.3.2,8.4.2,8.5.2,8.6.1,8.7.3,8.8.1,8.9.0-rc02,8.10.0-alpha05
+latests=8.4.2,8.5.2,8.6.1,8.7.3,8.8.2,8.9.0,8.10.0-beta01
 nightlyBuildId=13097777
 nightlyVersion=8.10.0-dev
```

Comment:
Tried to upgrade to `8.11.0-dev`, but there seem to be bugs in the current dev version causing us to run into [this bug](https://youtrack.jetbrains.com/issue/IDEA-369261) in our santa tracker smoke tests. 

So I chose to leave the nightly version as-is